backend/models/entry.js

```diff
@@ -0,0 +1,94 @@
+import Image from './image'
+import DataTypes from 'sequelize'
+import sequelize from '../config/sequelize'
+import { IMAGE_ENTRY } from '../constants'
+
+const Entry = sequelize.define('entry', {
+  showId: {
+    allowNull: false,
+    type: DataTypes.INTEGER
+  },
+  studentUsername: {
+    allowNull: true,
+    type: DataTypes.STRING
+  },
+  groupId: {
+    allowNull: true,
+    type: DataTypes.INTEGER
+  },
+  entryType: {
+    allowNull: false,
+    type: DataTypes.INTEGER
+  },
+  entryId: {
+    allowNull: false,
+    type: DataTypes.INTEGER
+  },
+  title: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  comment: {
+    type: DataTypes.TEXT
+  },
+  moreCopies: {
+    allowNull: false,
+    type: DataTypes.BOOLEAN
+  },
+  forSale: {
+    allowNull: false,
+    type: DataTypes.BOOLEAN
+  },
+  awardWon: {
+    type: DataTypes.TEXT
+  },
+  invited: {
+    allowNull: true,
+    type: DataTypes.BOOLEAN,
+    defaultValue: false
+  },
+  yearLevel: {
+    allowNull: true,
+    type: DataTypes.TEXT
+  },
+  academicProgram: {
+    allowNull: true,
+    type: DataTypes.TEXT
+  },
+  excludeFromJudging: {
+    allowNull: false,
+    type: DataTypes.BOOLEAN,
+    defaultValue: false
+  },
+  createdAt: {
+    allowNull: false,
+    type: DataTypes.DATE
+  },
+  updatedAt: {
+    allowNull: false,
+    type: DataTypes.DATE
+  }
+},
+{
+  validate: {
+    entrantPresentValidation () {
+      if (!this.studentUsername && !this.groupId) {
+        throw new Error('Entry must have an entrant')
+      }
+    }
+  }
+})
+
+/*
+ * Gets the associated photo as a Promise
+ */
+Entry.prototype.getImage = function getImage () {
```

Comment:
Could this be added in `instanceMethods` or does it have to be added to the prototype like this?